app/services/send_candidate_rejection_email.rb

```diff
@@ -0,0 +1,15 @@
+class SendCandidateRejectionEmail
+  def self.call(application_choice:)
+    candidate_application_choices = application_choice.application_form.application_choices
+    number_of_rejections = candidate_application_choices.select(&:rejected?).count
+
+    if candidate_application_choices.count == number_of_rejections
```

Comment:
You could just do `candidate_application_choices.all?(&:rejected?)` here I think. (minor/optional)

Reply:
Nice! Thanks